app/ui/src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts

```diff
@@ -263,17 +263,21 @@ export class DataMapperHostComponent extends FlowPage implements OnInit {
         this.initializeMapper();
       }
     });
-    this.countDownInitialization();
+    this.removeInitializationTask();
   }
 
   initializeMapper() {
-    if ( this.initializedCountDown == 0 ) {
+    if ( this.outstandingTasks == 0 ) {
```

Comment:
Mhhhh, apparently the linter did not automatically fixed this. Didn't you get an alert when running `yarn lint`?

Reply:
nope.  

```
$ yarn lint:fix
yarn run v1.2.1
$ yarn ng lint --fix


All files pass linting.
✨  Done in 9.70s.
```

Reply:
all CI checks passed. so I think your visual lint does not match up with how the linter is actually working.

Reply:
This is interesting: I do not use a visual lint.